source/mirror/rtti.d

```diff
@@ -254,9 +254,9 @@ abstract class Method {
     }
 
     immutable string identifier;
-    immutable RuntimeTypeInfo type;
+    const RuntimeTypeInfo type;
```

Comment:
This is a problem - the type info really should be immutable.

Reply:
Function `runtimeTypeInfo(T)` returns a mutable one (and builds the impl one). I guess it was a bug that it apparently used to be implicitly convertable.